srv/review-service.cds

```diff
@@ -27,4 +27,4 @@ service ReviewService {
     ];
 }
 
-annotate ReviewService.Reviews with @odata.draft.enabled;
+annotate Reviews with @odata.draft.enabled;
```

Comment:
I think here the qualifier is still required, as its defined outside the service -> see the test failure

Reply:
Interesting, that the compiler doesn't throw on this already during compilation..

Reply:
I think there is a typo: `ReviewsService` -> `ReviewService`, hm, sad that the compiler doesn't check this :(

Reply:
Yeah, I just saw it, too.